static/locales/fr/translation.json

```diff
@@ -152,5 +152,5 @@
 		"microsoft": "Microsoft",
 		"custom": "Personnalisé"
 	},
-	"translator_name": "Adofou"
-}
\ No newline at end of file
+	"translator_name": "GeekCorner"
```

Comment:
You weren't the sole translator, you provided updates. I'm looking into how we have added multiple credit before.